src/pykeen/models/unimodal/unstructured_model.py

```diff
@@ -19,7 +19,22 @@
 
 
 class UnstructuredModel(EntityEmbeddingModel):
-    """An implementation of Unstructured Model (UM) from [bordes2014]_."""
+    r"""An implementation of the Unstructured Model (UM) published by [bordes2014]_.
+
+    UM computes the distance between head and tail entities then applies the $l_p$ norm.
+
+    .. math::
+
+        f(h, r, t) = - \|\textbf{e}_h  - \textbf{e}_t\|_p^2
+
+    A small distance between the embeddings for the head and tail entity indicates a plausible triple. It is
+    appropriate for networks with a single relationship type that is undirected.
+
+    .. warning::
+
+        In UM, neither the relations nor the directionality are considered, so it can't distinguish between them.
```

Comment:
"cannot"?

Reply:
I don't mind using less formal wording in documentation